### DISCUSSION TITLE
Reading Sphingoid bases [SP01]

### DISCUSSION THREAD

**Discussion by JauntyJJS** [SysMedOs/LipidLynxX#24] [Q&A] [answered]
Hi,

I tried to use [LipidLynxX Converter](http://www.lipidmaps.org/lipidlynxx/converter/) to read some [Sphingoid bases[SP01]](https://www.lipidmaps.org/databases/lmsd/LMSP01010001?LMID=LMSP01010001)

![image](https://user-images.githubusercontent.com/9066508/175924058-7132a835-f1a1-42d1-a6e3-58711c016a26.png)

I was expecting the result to be SPB(18:1;O2) but the actual result I have is as follows

![image](https://user-images.githubusercontent.com/9066508/175924294-7659d748-4deb-40e7-a957-99849ba02140.png)

Could anyone verify if this output is valid or LipidLynxX Converter does not support sphingoid bases?

Thank you.

**Comment by ZhixuNi** [comment, marked as answer]
In LipidLynxX,we use () for compound with more than one possible residues e.g, phospholipids, TG, DG, SM, Cer...
FA and SPB are without ()

This is the current rule defined in the program.
You can make changes in the source code version to get FA() and SPB() but not via LipidMaps online version.

**Reply by JauntyJJS** [reply]
I see. Noted with thanks. I will take that "SPB18:1;O2" is a valid output then.

**Reply by ZhixuNi** [reply]
If you use source code version you can modify existed rules / add new rules in folder `/lynx/configurations/rules/output`.
You can try to add brackets/select the best output style as you like.

Best,

Ni
